lib/pact_broker/pacts/service.rb

```diff
@@ -42,6 +43,17 @@ def create_or_update_pact params
         end
       end
 
+      def merge_pact params
+        provider = pacticipant_repository.find_by_name_or_create params[:provider_name]
+        consumer = pacticipant_repository.find_by_name_or_create params[:consumer_name]
+        consumer_version = version_repository.find_by_pacticipant_id_and_number_or_create consumer.id, params[:consumer_version_number]
+        existing_pact = pact_repository.find_by_version_and_provider(consumer_version.id, provider.id)
+
+        params.merge!(json_content: Merger.merge_pacts(params[:json_content], existing_pact.json_content))
```

Comment:
hi @bethesque  Am I reading something wrong here or the parameters are written in wrong order?

 The `Merger.merge_pacts` method has a definition as follows:

```def merge_pacts original_json, additional_json```

so the original (existing_pact) should be first argument. Am I correct?

Reply:
Released in 2.27.3